.circleci/config.yml

```diff
@@ -93,12 +93,12 @@ jobs:
     executor: telegraf-ci
     steps:
       - checkout
-      - check-changed-files-or-halt
       - run: ./scripts/make_docs.sh
       - run: 'make deps'
       - run: 'make tidy'
       - run: 'make check'
       - run: 'make check-deps'
```

Comment:
Should these be run always or would it be better to halt before these if the necessary files aren't changed?

Reply:
Hmm I guess if the changes are only to markdown files then those four, deps/tidy/check/check-deps, don't really apply. Let me move it!